cypress.config.ts

```diff
@@ -33,7 +33,7 @@ export default defineConfig({
         reset: resetStubs,
         ...auth,
 
-        stubSignIn: () => auth.stubSignIn([]),
+        stubSignIn: (roles: []) => auth.stubSignIn(roles),
```

Comment:
The array was in the wrong place, and was preventing me passing in specific roles as an argument to the task in the new tests I've written